src/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,63 @@
+interface IconProps {
+  name: string;
```

Comment:
Let's create a type for the name so we get nice auto-complete, and the user gets a type error if they try and pass a non-valid name.